test/concourse/model/traits/filter_test.py

```diff
@@ -0,0 +1,140 @@
+import dataclasses
+import typing
+
+import pytest
+
+import concourse.model.traits.filter as examinee
+
+
+# Dummy dataclass for testing
+@dataclasses.dataclass
+class Dummy:
+    name: str
+
+
+@pytest.fixture
+def dummy():
```

Comment:
would it not be easier / less "boilerplaty" to simply add the default value to the dataclass-definition, and use its c'tor (/initialiser)? To me, a fixture makes more sense of it either offers some parametrisation (e.g. add additional attrs), or construction of returned object(s) is "cumbersome" (as in: requires multiple calls / operations)

Reply:
then again: we do we even need this dummy class?